samples/Makefile

```diff
@@ -1,26 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
-.PHONY: run
-
-define NEWLINE
-
-
-endef
-
-# Obtain list of installed samples.
-DIRS = $(wildcard */)
-
-build:
-	$(foreach i, $(DIRS), $(MAKE) -C $(i) $(NEWLINE) )
-
-clean:
-	$(foreach i, $(DIRS), $(MAKE) -C $(i) clean $(NEWLINE) )
-
-run-silent:
-	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) run $(NEWLINE) )
-
-run:
-	$(MAKE) -s run-silent
-
-world: clean build run
```

Comment:
I am curious why this Makefile is being removed?

Reply:
It was only needed by the old script to run the builds, but now we run the builds via `test-samples.cmake` which invokes `make` for each sample's directory.

Reply:
This PR deletes `samples/test-samples.sh`.

Reply:
@soccerGB had a really good point about checking the validation script in oe-engine, but we're good, the script uses each sample's `Makefile`, it never used this one.